### DISCUSSION TITLE
PR not reviewed properly

### DISCUSSION THREAD

**Discussion by hossein-zare** [laravel/framework#53273] [General] [answered]
Hello, Please review https://github.com/laravel/framework/pull/53222.

I submitted this PR days ago, but Taylor Otwell closed it with an odd reply, likely due to a misunderstanding. I don't think he even read what I wrote in the description or he checked the code i mean the changes.

I just want to ensure that the PR is properly reviewed, even if it's rejected.

Sorry for the inconvenience.

**Comment by crynobone** [comment, marked as answer]
You should submit a new PR if you wish Taylor to review it again.

**Reply by shaedrich** [reply]
Most likely, Taylor will close it again.

**Comment by shaedrich** [comment]
Word of advise: Don't submit PRs. What you experienced is not a mistake, it's intentionally. If you were Taylor's friend, he would approve your PR. But otherwise, chances are low that he does. For me, it's just not worth the effort, especially since you never know what his problem was this time.
